addon/components/cfb-form-editor/question.js

```diff
@@ -293,6 +268,12 @@ export default Component.extend(ComponentQueryManager, {
 
       optional([this.get("on-after-submit")])(question);
     } catch (e) {
+      const slug = changeset.get("slug");
+      const prefix = `${this.namespace}-`;
+      if (slug.startsWith(prefix)) {
+        changeset.set("slug", slug.replace(prefix, ""));
+      }
+
```

Comment:
Why is this part of the catch block? What's triggering the error?

Reply:
Ah, so you're resetting the change made earlier. I just pushed a commit that assigns the prefixed slug to a local variable instead of to the changeset. This has two advantages:

* no need to reset the value in case the save operation fails
* no visible "jump" from non-prefixed to prefixed slug in the user interface

Reply:
Nice! :sunglasses: